scripts/update-integrations.mjs

```diff
@@ -122,7 +126,9 @@ async function main() {
 			fs.rmSync(entry)
 		} else {
 			// fetch the latest NPM data, keeping any local overrides like description or icon
-			const details = await fetchWithOverrides(data.name)
+			// skipping download counts here since existing integrations will be updated
+			// automatically in a separate GitHub Action.
+			const details = await fetchWithOverrides(data.name, false)
```

Comment:
This is here to make sure the PR that requires manual review won't update every single integration with a new downloads counter